lib/spack/docs/packaging_guide.rst

```diff
@@ -1173,6 +1174,59 @@ you can check out a branch or tag by changing the URL. If you want to
 package multiple branches, simply add a ``svn`` argument to each
 version directive.
 
+.. _cvs-fetch:
+
+^^^
+CVS
+^^^
+
+CVS is an old centralized version control system similar to
+Subversion.
```

Comment:
I would say something like "Subversion is a fork of an older centralized version control system called CVS."

Reply:
Done.